src/help.html

```diff
@@ -127,172 +138,322 @@ <h2>Shipping FAQs</h2>
+        <li>Press the metal plate against the outside of the cage, ensuring the lever pokes inside between the bars. Check that the lever moves freely.</li>
         <li>
-          Screw wingnuts onto the bolts. Ensure the oval washer is horizontal and firmly tighten. Insert a microSD card with tunes into the slot and plug it into a USB port. Indicator LED should turn
-          green.
+          Hold the BeakBox in place and reach into the cage to insert the bolts, with oval washers attached, into the top and bottom mounting holes. Then, screw thumb nuts onto the bolts. Ensure the
+          oval washer is horizontal and firmly tighten.
         </li>
+        <li>Press the power button to turn your BeakBox on; the power light should turn green.</li>
       </ul>
-      <div>
-        <iframe
-          width="560"
-          height="315"
-          loading="lazy"
-          src="https://www.youtube.com/embed/_iG9LN9I2ug"
-          title="YouTube video showing how to install the Beakbox on the cage"
-          frameborder="0"
-          allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"
-          allowfullscreen
-        ></iframe>
+
+      <div class="notecard flex-container-collapse">
+        <img src="/imgs/warning-icon.svg" alt="">
+        <p>Parrots are often wary of new toys, so we strongly recommend gradually introducing your birds to the BeakBox before attaching it to the cage and teaching them how to press the lever.
+        </p>
       </div>
+  </article>
+</section>
 
-      <h3>How to connect my BeakBox to Wi-Fi</h3>
-      <p>To connect your BeakBox to Wi-Fi, you will need:</p>
+<section>
+  <article class="main-container main-container--narrow">
+    <div class="flex-container">
+      <h2 class="colored-text" id="beakbox-web-app">BeakBox web app</h2>
+      <p>With our BeakBox web app you can connect your BeakBox to your home Wi-Fi to receive future updates, upload your parrots' favorite music, create playlists, and remotely control your BeakBox!</p>
+    </div>
+    <div class="flex-container list-style-decimal">
+      <h3>How to access the BeakBox web app</h3>
+      <p>To access the BeakBox app, follow these steps:</p>
       <ul>
-        <li>A powered-on BeakBox</li>
-        <li>A device capable of connecting to Wi-Fi and browsing the internet (computer, tablet, mobile phone, etc.)</li>
+        <li>Turn on your BeakBox by pressing the power button; the LED light should turn green.</li>
+        <li>Press the Wi-Fi button; the LED light should start flashing blue.</li>
+        <li>
+          Open the Wi-Fi settings on your device and search for a network with the following name “MyBeakBox-NNNNNN” where NNNNNN is a set of six random numbers and characters. Use the password “boxflock“ to connect to your BeakBox.
+        </li>
+        <div class="notecard flex-container-collapse">
```

Comment:
The only permitted children of a `ul` are `li`, `script`, and `template` tags.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#technical_summary

Reply:
I'd also argue that this is an `ol` as the steps must be done in order.